### PR TITLE
bots: Use python3 in bootstrap script

### PR DESCRIPTION
**Pull request by marusak**
Now (since https://github.com/cockpit-project/cockpituous/commit/0fd4962c78a006dd22b628c84ce8bc95a51cf10e) tasks are run on F30, which does not support `python` binary anymore. To see if it works, lets refresh `rhel-atomic` - based on https://github.com/cockpit-project/cockpit/issues/11731

 * [x] image-refresh rhel-atomic

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on bos-tasks-h2590.
Log: https://209.132.184.41:8493/logs/image-refresh-11734-20190503-092727/

**Comment by cockpituous** [comment]
image-refresh in progress on bos-tasks-xt571.
Log: https://209.132.184.41:8493/logs/image-refresh-11734-20190503-092828/

File: bots/images/scripts/atomic.bootstrap [hunk 1]
```diff
@@ -65,7 +65,7 @@ fi
 # will grow the partitions etc as appropriate, and atomic.setup will
 # explicitly grow the docker pool.
 
-vsize=$(qemu-img info "$out" --output=json | python -c 'import json, sys; print json.load(sys.stdin)["virtual-size"]')
+vsize=$(qemu-img info "$out" --output=json | python3 -c 'import json, sys; print json.load(sys.stdin)["virtual-size"]')
```
Comment (martinpitt) [inline bots/images/scripts/atomic.bootstrap]: Thanks! FTR, to make this independent of Python one can also use `jq '."virtual-size"'`

**Comment by cockpituous** [comment]
Task failed: https://209.132.184.41:8493/logs/image-refresh-11734-20190503-092828/

**Comment by martinpitt** [comment]
```
    import json, sys; print json.load(sys.stdin)["virtual-size"]
                               ^
SyntaxError: invalid syntax
```

Heh, Python 3 print statement -- so please either enclose it in `()` (which will also work in Python 2), or use `jq`.

**Comment by cockpituous** [comment]
image-refresh in progress on bos-tasks-1qf8m.
Log: https://209.132.184.41:8493/logs/image-refresh-11734-20190504-075858/

**Comment by cockpituous** [comment]
image-refresh rhel-atomic done: https://github.com/cockpituous/cockpit/commits/image-refresh-rhel-atomic-20190504-092828

**Comment by marusak** [comment]
Ah, I forgot the Closes #... line, so I need to repush. For the record, all tests passed. I'll try to use the push tool, lets see if it keeps tests.

**Comment by marusak** [comment]
No, didn't work. Most likely because it makes it as no-test with label, but we know the label does not work in fast sequences (and possibly now is broken all-together). Would be better if the push-rewrite tool would use [no-test] in title, that seems to work better.

**Pull request merged**